tests/encodehelp.h

```diff
@@ -152,13 +155,16 @@ static bool process_cmdline(int argc, char *argv[])
                     rcMode = string_to_rc_mode(optarg);
                     break;
                 case 3:
-                    ipbMode= atoi(optarg);
+                    ipbMode = atoi(optarg);
```

Comment:
Thanks for your comments. Actually I didn’t use git-format-clang. I just noticed this line in irregular format and then changed it incidentally.

Best Regards,
Xiaoxia Zhong

From: Xu Guangxin [mailto:notifications@github.com]
Sent: Monday, February 29, 2016 10:09 AM
To: 01org/libyami
Cc: Zhong, Xiaoxia
Subject: Re: [libyami] Add IDR interval setting to H.264 encoder (#447)


In tests/encodehelp.h<https://github.com/01org/libyami/pull/447#discussion_r54363499>:

> @@ -152,13 +155,16 @@ static bool process_cmdline(int argc, char *argv[])

>                      rcMode = string_to_rc_mode(optarg);

>                      break;

>                  case 3:

> -                    ipbMode= atoi(optarg);

> +                    ipbMode = atoi(optarg);

This changed by git-format-clang,right? Instead of format your file you can format your patch only. git-format-clang --sytle=webrtc HEAD^. Use this approach we can find who is really change the code by "git blame"

—
Reply to this email directly or view it on GitHub<https://github.com/01org/libyami/pull/447/files#r54363499>.